_data/tool_and_resource_list.yml

```diff
@@ -2660,3 +2660,48 @@
   id: better-bibtex
   name: Better BibTeX (BBT)
   url: https://retorque.re/zotero-better-bibtex/
+- description: GA4GH frames policy and builds standards to meet the real-world needs of the global genomics and health community.
+  id: ga4gh
+  name: Global Alliance for Genomics and Health (GA4GH)
+  url: https://www.ga4gh.org
+  registry:
+    biotools: ga4gh
+    fairsharing: 2tpx4v
```

Comment:
I am not sure where this is coming from, but this leads to a deprecated entry https://fairsharing.org/FAIRsharing.2tpx4v that we probably should not indicate. Note also that the entry you created in the table if for an organisation, whereas the (deprecated) FAIRsharing page points to an output of that organisation, not to the organisation itself.

Reply:
Same, I think I also took this one over from IDTk, means I should have a look at the records there too